docs/stack/subgraph/queries.md

```diff
@@ -0,0 +1,346 @@
+You can use GraphQL queries to get quick info from [DAOstack Sunbgraph](https://thegraph.com/explorer/subgraph/daostack/v30_1) hosted on GraphExplorer.
+
+This guide explains how to use GraphQL queries to get single or multiple Entities and sort, filter and paginate them.
+
+The full list of `Subgraph Entities` cached by DAOstack subgraph can be found [here](../Entity)
+
+## General Guidelines
+
+  1. All queries must be wrapped inside `query {}` object.
+  2. Entity name is same as provided in [Entity list](../Entity) but in `lower case`.
+  3. You can [query for single entity](#query-for-single-entity) by providing Entity `id`.
+  4. You can [query for multiple Entities](#query-for-multiple-entities) by changing entity to plural. i.e. `proposal` -> `proposals`   
+  5. The `complex-field` i.e. fields that are themselves an Entity such as *dao* and *proposals* in above example, need to be wrapped in `{}` and provided with the subfield needed to be queried
+  6. Filter and Sort is not available for complex fields
```

Comment:
This is not correct anymore. It used to not be available but it now is. See query like this for example:
```
{
  daos {
    proposals(first: 2) {
      id
    }
  }
}
```

Reply:
Wow this is so awesome. Didn't know about it. Will add complex filter and sort examples here too

Reply:
Ahh but we still can't filter based on complex fields.
Like if I want to get details of only those DAOs that has schemes' `version_gt: x`, I still can't do that in single query....can I?

Reply:
Yes filtering based on such complex fields (basically filtering by fields of fields) is not supported.